keyboards/anavi/knobs3/knobs3.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2022 Leon Anavi <leon@anavi.org>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+// clang-format off
+ #define LAYOUT_k3(  \
+   K01, K02, K03 \
+ ) \
+ { \
+   { K01, K02, K03 } \
+ }
+// clang-format on
```

Comment:
This file can be deleted with the above changes to info.json.

Reply:
Thank you for the recommendations. I have applied the changes and forced pushed the amended git commit.